full-moon/src/ast/mod.rs

```diff
@@ -413,11 +430,11 @@ pub enum Stmt {
 
     /// A goto statement, such as `goto label`
     /// Only available when the "lua52" or "luajit" feature flag is enabled.
-    #[cfg(any(feature = "lua52", feature = "luajit"))]
+    #[cfg(any(feature = "lua52", feature = "luajit", feature = "cfxlua"))]
```

Comment:
Same as above and in the other cases, I think this can be left alone
```suggestion
    #[cfg(any(feature = "lua52", feature = "luajit"))]
```